src/component/AddTodoForm.js

```diff
@@ -24,8 +22,8 @@ function AddTodoForm(props) {
           name="title" required
           id="todoTitle"
           placeholder="Add new todo"
-          onChange={e => setInputVal(e.target.value)}
-          value={inputVal}
+          value={initText}
+          onChange={e => setInitText(e.target.value)}
```

Comment:
Assignment **lesson-1-3** doesn't require using `onChange`, but it's good approach. Well done!